<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBindDialog</class>
 <widget class="QDialog" name="KeyBindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка &quot;горячих клавиш&quot;</string>
  </property>
  <widget class="QComboBox" name="cbAction">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Действие:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Комбинация клавиш</string>
   </property>
   <widget class="QComboBox" name="cbKey">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMod1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbMod2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Модификатор 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Модификатор 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Клавиша-ключ:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
